docs/docs/usage_instructions/restoring_purchases.md

```diff
@@ -8,37 +8,41 @@ You can use `getAvailablePurchases()` to do what's commonly understood as “res
 
 If for debugging you want to consume all items, you have to iterate over the purchases returned by `getAvailablePurchases()`. Beware that if you consume an item without having recorded the purchase in your database the user may have paid for something without getting it delivered and you will have no way to recover the receipt to validate and restore their purchase.
 
-```javascript
-  getPurchases = async () => {
-    try {
-      const purchases = await RNIap.getAvailablePurchases();
-      const newState = { premium: false, ads: true }
-      let restoredTitles = [];
-
-      purchases.forEach(purchase => {
-        switch (purchase.productId) {
+```ts
+getPurchases = async () => {
+  try {
+    const purchases = await RNIap.getAvailablePurchases();
+    const newState = {premium: false, ads: true};
+    let restoredTitles = [];
+
+    purchases.forEach((purchase) => {
+      switch (purchase.productId) {
         case 'com.example.premium':
-          newState.premium = true
+          newState.premium = true;
           restoredTitles.push('Premium Version');
-          break
+          break;
 
         case 'com.example.no_ads':
-          newState.ads = false
+          newState.ads = false;
           restoredTitles.push('No Ads');
-          break
+          break;
 
         case 'com.example.coins100':
-          await RNIap.finishTransaction(purchase.purchaseToken);
+          await RNIap.consumePurchaseAndroid(purchase.purchaseToken);
```

Comment:
This should be `finishTransaction`

Reply:
Thanks, not sure how this got change here